static/helpers.js

```diff
@@ -1,13 +1,24 @@
 /* eslint no-unused-vars: 1 */
 /* eslint no-new-func: 0 */
 /* *global __rootDirectory */
-;(function (global) {
+; (function (global) {
   const Handlebars = require('handlebars')
 
   global.docxPageBreak = function () {
     return new Handlebars.SafeString('')
   }
 
+  global.docxRaw = function (data) {
+    if (typeof data === 'string') {
+      if (data.startsWith('<w:r')) {
+        return new Handlebars.SafeString(data)
+      }
+    }
+
+    // Wrap not valid XML data as a literal, without any style
+    return new Handlebars.SafeString('<w:r><w:t>' + data + '</w:t></w:r>')

```

Comment:
We probably need to adapt this to the value of `replaceParentElement` https://github.com/jsreport/jsreport-docx/pull/23/files#r518899344